Source/Scene/Points3DTileContent.js

```diff
@@ -340,12 +423,80 @@ define([
         var positions = parsedContent.positions;
         var colors = parsedContent.colors;
         var normals = parsedContent.normals;
+        var batchIds = parsedContent.batchIds;
         var isQuantized = parsedContent.isQuantized;
         var isOctEncoded16P = parsedContent.isOctEncoded16P;
+        var styleableProperties = parsedContent.styleableProperties;
         var isTranslucent = content._isTranslucent;
 
         var hasColors = defined(colors);
         var hasNormals = defined(normals);
+        var hasBatchIds = defined(batchIds);
+
+        var batchTable = content.batchTable;
+        var hasBatchTable = defined(batchTable);
+        var hasStyleableProperties = defined(styleableProperties);
+
+        // Use per-point normals to hide back-facing points.
+        var backFaceCulling = true;
```

Comment:
Also add a TODO here for how to expose this, e.g., will the Cesium API have a point cloud specific style, is this a tileset property, etc.